BunnymodXT/modules/ServerDLL.cpp

```diff
@@ -1520,6 +1521,8 @@ HOOK_DEF_1(ServerDLL, void, __cdecl, PM_PlayerMove, qboolean, server)
 	}
 
 	CustomHud::UpdatePlayerInfo(velocity, origin);
+
+	not_stuck_prev_frame = stuck_cur_frame ? false : true;
```

Comment:
```suggestion
	not_stuck_prev_frame = !stuck_cur_frame;
```

Would've saved some trouble if you made it `stuck_prev_frame` rather than `not_stuck_prev_frame`